vendor.conf

```diff
@@ -6,28 +6,29 @@ github.com/davecgh/go-spew v1.1.0
 github.com/pmezard/go-difflib v1.0.0
 golang.org/x/sys 314a259e304ff91bd6985da2a7149bbf91237993
 
-github.com/containerd/containerd 8c2acf45fb50eec264aef469ed2f42e9b0cfc865
+github.com/containerd/containerd a15e7a0be093db114409d377dcee75c4ddce5e34
 github.com/containerd/typeurl f6943554a7e7e88b3c14aad190bf05932da84788
 golang.org/x/sync 450f422ab23cf9881c94e2db30cac0eb1b7cf80c
 github.com/sirupsen/logrus v1.0.0
-google.golang.org/grpc v1.7.4
+google.golang.org/grpc v1.10.1
 github.com/opencontainers/go-digest c9281466c8b2f606084ac71339773efd177436e7
-golang.org/x/net 0ed95abb35c445290478a5348a7b38bb154135fd
-github.com/gogo/protobuf v0.5
+golang.org/x/net b3756b4b77d7b13260a0a2ec658753cf48922eac
```

Comment:
This one is a downgrade. `0ed95abb35c445290478a5348a7b38bb154135fd` is from `go1.10` feature branch. I guess we should leave it as is in buildkit and upgrade containerd to correct version instead.

Reply:
reverted